libs/components/popovers/src/lib/modules/popover/popover.directive.ts

```diff
@@ -220,6 +236,22 @@ export class SkyPopoverDirective implements OnInit, OnDestroy {
   }
 
   private sendMessage(messageType: SkyPopoverMessageType): void {
-    this.skyPopoverMessageStream.next({ type: messageType });
+    this.messageStream.next({ type: messageType });
+  }
+
+  private setupMessageStream(value?: Subject<SkyPopoverMessage>): void {
+    this.teardownMessageStream();
+
+    this._messageStream = value ?? new Subject<SkyPopoverMessage>();
```

Comment:
I generally try to avoid referencing any private fields that back getter/setters in anything other than the getter/setter. That ensures all access (even within the same class) goes through the getter, which sometimes may have logic of its own, like returning default values. I'd move the assignment here back into the setter, then change `_messageStreamSub` to `messageStreamSub` (or better yet, `#messageStreamSub` to make it truly private) since it doesn't back a getter/setter property.